### DISCUSSION TITLE
Environment variable to disable the logs to false across Cypress run

### DISCUSSION THREAD

**Discussion by karanshah-browserstack** [cypress-io/cypress#16936] [Questions and Help]
Hi, 
Is there any flag/code where we can disable logging of commands across the whole cypress run? 
https://docs.cypress.io/api/commands/type#Arguments
We can do it by `cy.get('yourelement').type('password',{log:false});`.

**Comment by jennifer-shehane** [comment]
Are you trying to disable the logging for all of a specific command? Like - don't log any `.type()` commands? If so, you could overwrite the command and pass `log: false` to the overwrite. 

Here's an example of overwriting the type command so that it will not show passwords for example: https://on.cypress.io/custom-commands#Overwrite-type-command

If you want to disable all command logging, this isn't possible. 

Can I ask why you're looking to disable all the logs?

**Reply by karanshah-browserstack** [reply]
@jennifer-shehane Yes, we want to disable all the command logging instead of manually overwriting every type of command which we use across over project. 
Use-case is to disable all the type commands, as it may contain some sensitive information coming in logs/ video which we need to share among the teams internally.

**Reply by jennifer-shehane** [reply]
Ok, the only solution is as shown above. Overwrite the type command - you can place this code in the `supportFile` and it will apply to every `type` test in the project.
